### PR TITLE
Apply latest nlm generator

### PR DESCRIPTION
**Pull request by markowsiak**
*This PR was created by a machine.*

[`nlm`](https://github.com/groupon/nlm) is a tool for automating the release of node packages. There are some notable implications of using `nlm`:

* Commits messages are used to determine the semver implications of the changes.
* The above means that all commits have to follow the [angular commit message conventions](https://gist.github.com/stephenparish/9941e89d80e2bc58a153) so `nlm` can parse them.
* Releases happen automatically from CI after successful builds of the master branch. There's no manual intervention required.

Additionally some standard dependency versions were enforced:

* `bluebird@^3.3.3`
* `lodash@^4.6.1`

/cc @groupon/itier

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,12 +2,16 @@
   "name": "shared-store",
   "version": "2.2.1",
   "description": "Keeping config data in sync",
-  "license": "BSD-3-Clause",
+  "license": {
```
Comment (jkrems) [inline package.json]: This seems wrong..?

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-  - 'git config --global user.email "opensource@groupon.com"'
-  - 'git config --global user.name "Groupon"'
-deploy:
-  provider: script
-  script: ./node_modules/.bin/nlm release
-  skip_cleanup: true
-  'on':
-    branch: master
-    node: '4'
+  - git config --global user.email "opensource@groupon.com"
```
Comment (jkrems) [inline .travis.yml]: For `gofer` these lines are gone - do we know why they stuck around here?

Comment (markowsiak) [reply .travis.yml]: Looking back, the generator update only removed their presence in default config etc, and did not handle their removal.  I've been removing them manually as I've come across them.

Comment (jkrems) [reply .travis.yml]: Ah, interesting. Not sure if it's worth adding to the generator. Hopefully there aren't that many still around (and it's not like they hurt anything).

**Comment by markowsiak** [comment]
Nice catches @jkrems . Ready for another look 👍

**Pull request merged**